fpga_interchange/examples/devices/xc7a35t/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+generate_xc7_device_db(
+    device xc7a35t
+    part xc7a35tcsg324-1
+    device_target target
+)
+
+generate_chipdb(
+    family ${family}
+    device xc7a35t
+    part xc7a35tcsg324-1
+    device_target ${target}
+    bel_bucket_seeds ${PYTHON_INTERCHANGE_PATH}/test_data/series7_bel_buckets.yaml
+    package csg324
```

Comment:
The package here is only used for testing the chipdb.  Maybe rename the argument to "test_package" and mention this in the README.  The key here is that this chipdb has more packages than just csg324, and this looks confusing right now.

Reply:
Done